pkg/apis/extensions/types.go

```diff
@@ -87,6 +88,18 @@ type CPUTargetUtilization struct {
 	TargetPercentage int `json:"targetPercentage"`
 }
 
+// Alpha-level support for Custom Metrics in HPA (as annotations).
+type CustomMetricUtilization struct {
```

Comment:
Also, avoiding floating point numbers would be great.  So, if this going to be a Utilization, then can it at least be a PercentUtilization?